src/main/kotlin/uk/gov/dluhc/printapi/database/entity/Certificate.kt

```diff
@@ -81,6 +81,11 @@ class Certificate(
      */
     var initialRetentionRemovalDate: LocalDate? = null,
 
+    /**
+     * Set to true after the initial retention period data is removed.
+     */
+    var initialRetentionDataRemoved: Boolean = false,
```

Comment:
Does this need to be `Boolean?` - asking as noticed the Liquibase column allows nulls.

Reply:
Made column non-nullable.